### PR TITLE
Moving shared OD/Live radio styling to one place

### LINKED ISSUE #7440: Use of '!important' causing the AMP validator to fail
**Describe the bug**
**The amp:validate check is failing on OD radio (`/indonesia/bbc_indonesian_radio/w172xh267fpn19l`) & live radio (`/amharic/bbc_amharic_radio/liveradio`) because of a use of `!important`. This is not causing the build to fail.**

Additionally it is failing on `mundo/23263889`

```
/amharic/bbc_amharic_radio/liveradio
FAIL
line 25, col 36: The text inside tag 'style amp-custom' contains 'CSS !important', which is disallowed. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#stylesheets)
/indonesia/bbc_indonesian_radio/w172xh267fpn19l
FAIL
line 35, col 36: The text inside tag 'style amp-custom' contains 'CSS !important', which is disallowed. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#stylesheets)
/mundo/23263889
FAIL
line 1, col 7330: The author stylesheet specified in tag 'style amp-custom' is too long - document contains 80113 bytes whereas the limit is 75000 bytes. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#maximum-size)
```

**Due to this not being a check that stops the build, a change to the styles to liveradio pages were made which invalidated them as AMP pages and this was not highlighted in CI**
Currently, Liveradio AMP pages are not valid:
```
The text inside tag 'style amp-custom' contains 'CSS !important', which is disallowed.
```
https://validator.ampproject.org/#url=https%3A%2F%2Fwww.bbc.com%2Famharic%2Fbbc_amharic_radio%2Fliveradio.amp

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to a Jenkins build for a PR to 'latest'.
2. Look at the stage `make developmentTests`
3. Scroll down to 'FAIL'

**Expected behaviour**
- Issues with the relevant AMP pages are addressed (e.g. the CSS !important usage). 
- **(TO BE DISCUSSED - Should AMP validator fail the build?)** AMP validate is running and stopping the build in a visible way.

**Screenshots**
Screenshot of Jenkins
Screenshot of Jenkins

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #7440

**Overall change:**
The scrub bar location icon (the red dot) is cut off at the start of an episode. Instead of adding the fix in both OD Radio and Live Radio, I've moved the styling to one place and fixed the issue there.

**Code changes:**

- Added margin for amp-iframe
- Added div > iframe for the iframe query. This is because the AMP page has 3 nested iframes (amp-iframe -> iframe -> SMP iframe). The styling specified in iframe{} was therefore being applied to the second nested iframe on AMP pages. Narrowed the query to only apply the styling to the iframe that is a direct child of a div (as per canonical pages). The second AMP iframe is a child of the amp-iframe and so this styling is no longer applied.
- We still need to keep the iframe{}/amp-iframe{} queries to prevent the styling being applied to the No-JS message

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/integration/pages/frontPage/arabic/__snapshots__/amp.test.js.snap [hunk 1]
```diff
@@ -832,6 +832,7 @@ exports[`Story Promo Image should match image 1`] = `
   height="549"
   layout="responsive"
   src="https://ichef.bbci.co.uk/news/660/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg"
+  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/17FC4/production/_107844289_055122086-1.jpg 660w"
```
Comment (karinathomasbbc) [inline src/integration/pages/frontPage/arabic/__snapshots__/amp.test.js.snap]: Can't see why this snapshot has changed - seems unrelated to your changes, but to RPK's image component PR instead?

File: src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap [hunk 2]
```diff
@@ -143,6 +143,26 @@ Object {
 }
 `;
 
+exports[`Canonical Media Asset Page I can see related content: "المايوه الشرعي" يثير جدلا في الساحل الشمالي في مصر 1`] = `"/arabic/tv-and-radio-53492142"`;
```
Comment (karinathomasbbc) [inline src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap]: Similarly, I don't know why these snapshots are here, these are related content / most watched. Think this ought to be fixed separately, and not part of this PR

Comment (rebeccamcginn) [reply src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap]: Integration snapshots are failing on latest; everyone else seems to be discarding the updates

Comment (karinathomasbbc) [reply src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap]: There's definitely something weird going on - I thought Andrew fixed them last week, so don't know why they are coming back.

**Comment by rebeccamcginn** [comment]
Updating integration snapshots here: https://github.com/bbc/simorgh/pull/7703

File: src/app/containers/AVPlayer/StyledAudioPlayer.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import AVPlayer from '.';
+
+const StyledAudioPlayer = styled(AVPlayer)`
+  amp-iframe {
```
Comment (jroebu14) [inline src/app/containers/AVPlayer/StyledAudioPlayer.jsx]: you can just do 
```suggestion
  amp-iframe, div > iframe {
```

and remove the other one if they're the same. also not sure about having this as a separate importable component

**Review by jroebu14 — APPROVED** [review]

**Review by RichardPK — APPROVED** [review]
